### PR TITLE
Updated documentation for migrate.sh script

### PR DESCRIPTION
**Pull request by singingwolfboy**
@mhoeber or @lamagnifica, can one of you review this?

### CODE REVIEW

File: en_us/install_operations/source/birch.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ What's Included in Birch
 The Open edX Birch release contains several new features for students, course
 staff, and developers.  See the Open edX Release Notes for more details. (LINK)
```
Comment (lamagnifica) [inline en_us/install_operations/source/birch.rst]: please comment out (LINK) for the time being

Comment (singingwolfboy) [inline en_us/install_operations/source/birch.rst]: I figured it would be simplest just to delete that text entirely.

**Comment by mhoeber** [comment]
:+1:

**Comment by lamagnifica** [comment]
:+1:

**Pull request merged**